### PR TITLE
Add documentation for customer order api endpoint

### PR DESCRIPTION
**Pull request by ionut998**

### CODE REVIEW

**Comment by reevoo-samuel** [comment]
*If you're new, read the [guidelines](https://github.com/reevoo/guidelines/blob/master/pull_requests.md).*

Good code is **tested**, **easy to understand** and **able to cope when things go wrong**.

Good pull requests are **descriptive**, **small** and **short-lived**.

- [ ] Code has been reviewed.
- [ ] Description explains what you have done and why.
- [ ] Tests are green on CI (or pulled/tested on another machine).
- [ ] Documentation has been created or updated.

**Comment by lpil** [comment]
I feel `uuid` and `trkref` should be in caps, as they are in the responses from the API.

File: docs/reevooapi/customer-order/customer-order-batch-submission.markdown [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+id: customer-order-batch-submission
+layout: docs
+title: Platform API v4 / Customer Order Batch Submission
+navId: docs
+group: reevooapi
+prev: customer-order/customer-order-single-submission
+---
+
+# Customer Order Batch Submission
+This endpoint enables the submission of a batch of customer orders data. Not all customer orders in the batch need to be associated to the same organisation trkref.
```
Comment (lpil) [inline docs/reevooapi/customer-order/customer-order-batch-submission.markdown]: Like here

**Comment by ionut998** [comment]
They are in caps now @lpil

**Comment by lpil** [comment]
Hi Alexandru! :D

**Comment by jesuslarareevoo** [comment]
This is good.

**Pull request merged**